central/notifiers/cscc/cscc_test.go

```diff
@@ -0,0 +1,87 @@
+package cscc
+
+import (
+	"context"
+	"testing"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/golang/mock/gomock"
+	clusterMocks "github.com/stackrox/rox/central/cluster/datastore/mocks"
+	"github.com/stackrox/rox/central/notifiers/cscc/findings"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWithFakeCSCC(t *testing.T) {
+
+	const (
+		projectKey = "FJ"
+	)
+
+	sourceID := "organizations/847401270788/sources/394194234514334994"
```

Comment:
Are these random numbers in the sourceID or are they for an actual source?

Reply:
Please use fake data